README.md

```diff
@@ -8,18 +8,36 @@ Make sure you're running a local ILP provider. If you don't have one, take a loo
 ```sh
 npm install -g ilp-spsp
 # payment pointers use the '$' so must be escaped in bash
+
 ilp-spsp --help
-ilp-spsp send --receiver \$sharafian.com --amount 1000
-ilp-spsp query --receiver \$sharafian.com
-# --> {
-#   "destinationAccount": "g.siren.out.XOj2Y_doFnoFDZTp0sKKvUOfw7XFpW9OyWyz7gkxQhE.FOiDpSe0xmo.UpGrSQGr7TvDmnLUoEez0Gzg",
+# --> 
+# ilp-spsp [command]
+
+# Commands:
+#   ilp-spsp send     send money via SPSP
+#   ilp-spsp invoice  pay an SPSP invoice
+#   ilp-spsp pull     pull money via SPSP
+#   ilp-spsp query    query SPSP endpoint
+
+# Options:
+#   --help                         Show help                                             [boolean]
+#   --version                      Show version number                                   [boolean]
+#   --pointer, --receiver, -p, -r  SPSP payment pointer
+
+ilp-spsp query --pointer '$mysubdomain.localtunnel.me'
+# --> 
+# {
+#   "destinationAccount": "private.moneyd.local.DMMyURutr6hUmF-Go0ch3SAnsvrKKnmqG6oWwtEDjTA.cFKMLZPFaXJCL_pqqjLtLWPE~ccf42f59-5692-4a8b-8fde-4896c7601035",
 #   "sharedSecret": "qtUsvKOjVX9b6WIZT5KVTJhXnkej0P5kH6vAoRm9atU=",
-#   "ledgerInfo": {
-#     "asset_code": "XRP",
-#     "asset_scale": 6
-#   },
-#   "receiverInfo": {
-#     "name": "Ben Sharafian"
-#   }
-# }
+
+ilp-spsp send --pointer '$mysubdomain.localtunnel.me' --amount 1000
+# --> 
+# paying 100 to "$mysubdomain.localtunnel.me"...
+# sent!
+
+ilp-spsp pull --pointer '$mysubdomain.localtunnel.me/f8095a44-c77f-4414-a19d-7aeca03f17c7' -- amount 100
+# --> 
+# pulled 100 units! 
 ```
+
+The amount in the pull command is optional. If it is not provided, the maximum possible amount will be pulled, depending on the pull payment agreement. 
```

Comment:
I imagine there are probably cases where this makes sense, and I realize there's nothing stopping someone from using their own implementation to pull money.
That said, I'm a little uneasy about making amount optional (and likewise making `Inifinity` the default amount https://github.com/interledgerjs/ilp-protocol-spsp/pull/4/files#diff-168726dbe96b3ce427e7fedce31bb0bcR105) making it as easy as possible for someone to pull the maximum possible amount.

Reply:
There is no way for the pulling entity to know how much they can pull because the fields in the endpoint query response are all optional. I guess you can argue that the pulling entity should remember what they negotiated when they requested a token/endpoint and would thus know how much they are able to pull. 
Is it really harmful to set the receiveMax to Infinity because there is always a sendMax less than Infinity on the server side?

Reply:
I think I'd prefer a payee be able to use this tooling to pull a specific amount (which could for whatever reason exceed `sendMax`, in which case they would receive the `sendMax` amount).
I don't think it should offer a convenient way for a payee to simply receive the `sendMax` amount.
But like I said, I'm probably overlooking use cases and realize someone could set `receiveTotal` to `Infinity` themselves.